test/end-to-end/test_studio_auto_installs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
```

Comment:
If we can try and rely as much as possible on standard environment variables here, I think that would be easier to reason about, as well as easier to run locally (fewer "special" variables to mentally juggle).

To that end, relying on `HAB_BLDR_URL` already being set from outside would be nice to do if we can; I know I ran into some situations where code that runs before the test was trying to download expeditor itself based on this variable, and ultimately failing. If that's _not_ the case here, simplifying would be nice.

Reply:
I've tried adding HAB_BLDR_URL to the `environment` hash in the pipeline definition and it appears to have no effect, so there is something wrong with how we're using that that I need to dig into.

Reply:
Turns out I can't spell `acceptance` :man_facepalming: